ingest/Snakefile

```diff
@@ -92,13 +93,19 @@ rule fetch_ncbi_dataset_package:
         taxon_id=TAXON_ID,
         api_key=NCBI_API_KEY,
         gateway=f"--gateway-url {NCBI_GATEWAY_URL}" if NCBI_GATEWAY_URL else "",
+        use_mirror=bool(MIRROR_BUCKET), 
+        mirror_bucket=MIRROR_BUCKET,
```

Comment:
I think it's clearer to construct the URL here as one doesn't actually see the lack of space in review mode:
<img width="543" alt="image" src="https://github.com/user-attachments/assets/cc50ca2e-564e-4525-b825-b5e31ae1869b" />
```suggestion
        mirror_url=lambda w: f"{MIRROR_BUCKET}.strip('/')}/{TAXON_ID}.zip"
```

Reply:
Stripping means it doesn't need to end with trailing `/`, support both with and without trailing `/`